src/lib/theme/plugin.ts

```diff
@@ -1,122 +1,13 @@
-  '--color-border-interactive-secondary': rgb(getColor('slate', 100)),
-
-  '--color-shadow-primary': rgb(getColor('indigo', 600)),
-  '--color-shadow-secondary': rgb(getColor('indigo', 500)),
-  '--color-shadow-danger': rgb(getColor('red', 200)),
-} as const satisfies Variables;
-
-const dark: Partial<Variables<keyof typeof variables>> = {
-  '--color-primary': rgb(colors.offWhite),
-  '--color-secondary': rgb(colors.offBlack),
-  '--color-inverse': rgb(colors.primary),
-  '--color-brand': rgb(getColor('indigo', 500)),
-  '--color-interactive-error': rgb(getColor('red', 400)),
-  '--color-interactive-active': rgb(getColor('indigo', 500)),
-  '--color-warning': rgb(getColor('yellow', 950)),
-
-  '--color-text-primary': rgb(colors.offWhite),
-  '--color-text-secondary': rgb(getColor('slate', 200)),
-  '--color-text-subtle': rgb(getColor('slate', 200)),
-  '--color-text-disabled': rgb(getColor('slate', 100)),
-  '--color-text-information': rgb(getColor('blue', 300)),
-  '--color-text-warning': rgb(getColor('yellow', 300)),
-  '--color-text-error': rgb(getColor('red', 100)),
-  '--color-text-success': rgb(getColor('green', 100)),
-  '--color-text-active': rgb(getColor('indigo', 500)),
-
-  '--color-surface-background': rgb(colors.offBlack),
-  '--color-surface-primary': rgb(colors.black),
-  '--color-surface-secondary': rgb(colors.offBlack),
-  '--color-surface-interactive-secondary': rgb(getColor('slate', 800)),
-  '--color-surface-subtle': rgb(getColor('slate', 900)),
-  '--color-surface-table': rgb(getColor('slate', 900)),
-  '--color-surface-badge': rgb(getColor('slate', 700)),
-  '--color-surface-information': rgb(getColor('indigo', 950)),
-  '--color-surface-warning': rgb(getColor('yellow', 950)),
-  '--color-surface-danger': rgb(getColor('red', 300)),
-  '--color-surface-error': rgb(getColor('red', 950)),
-  '--color-surface-success': rgb(getColor('green', 950)),
-
-  '--color-border-primary': rgb(getColor('slate', 600)),
-  '--color-border-secondary': rgb(getColor('slate', 700)),
-  '--color-border-subtle': rgb(getColor('slate', 800)),
-  '--color-border-table': rgb(getColor('slate', 900)),
-  '--color-border-inverse': rgb(colors.black),
-  '--color-border-disabled': rgb(getColor('slate', 100)),
-  '--color-border-information': rgb(getColor('blue', 700)),
-  '--color-border-interactive': rgb(getColor('indigo', 600)),
-  '--color-border-warning': rgb(getColor('yellow', 700)),
-  '--color-border-danger': rgb(getColor('red', 300)),
-  '--color-border-error': rgb(getColor('red', 700)),
-  '--color-border-success': rgb(getColor('green', 700)),
-  '--color-border-interactive-secondary': rgb(getColor('slate', 800)),
-
-  '--color-shadow-danger': rgb(getColor('red', 600)),
-} as const;
+import { colors } from './colors';
+import { css } from './utilities';
+import { dark, light } from './variables';
 
 const temporal = plugin(
   ({ addComponents, addBase }) => {
     addBase({
-      ':root': variables,
+      ':root': light,
```

Comment:
Something is off... Noticing alerts, the table header, side nav, and various other colors aren't showing as expected in day/night modes ⬇️ 

| Before | After |
|--|--|
|![Screenshot 2024-05-13 at 12 10 40 PM](https://github.com/temporalio/ui/assets/15069288/20fc4668-497b-46d4-8559-17b680cf3b01)|<img width="962" alt="Screenshot 2024-05-13 at 12 10 48 PM" src="https://github.com/temporalio/ui/assets/15069288/fe766eaa-ed9c-4b17-9289-7811e8570de9">|

<img width="1240" alt="Screenshot 2024-05-13 at 12 14 43 PM" src="https://github.com/temporalio/ui/assets/15069288/30346001-6029-4e90-b2c1-c555cf3f15f4">